infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/send/SnapshotSender.java

```diff
@@ -122,7 +122,7 @@ public void transmit(UUID snapshotSyncEventId, boolean forcedSnapshotSync) {
         if (Address.isAddress(baseSnapshotTimestamp)) {
             // Read and Send Batch Size messages, unless snapshot is completed before (endRead)
             // or snapshot sync is stopped
-            dataSenderBufferManager.resend();
+            dataSenderBufferManager.resend(false);
```

Comment:
For future reference, can you add a comment stating why this is enabled for log entry sync only?